Microsoft.Identity.Web/Client/TokenAcquisition.cs

```diff
@@ -223,7 +229,7 @@ public async Task<string> GetAccessTokenOnBehalfOfUser(HttpContext context, IEnu
         /// }
         /// </code>
         /// </example>
-        public void AddAccountToCacheFromJwt(AspNetCore.Authentication.OpenIdConnect.TokenValidatedContext tokenValidatedContext, IEnumerable<string> scopes = null)
```

Comment:
This class uses both the OpenIdConnect and the JwtBearer middleware. 
I believe that the TokenValidatedContext is different?